cmd/deploy/endpoints.go

```diff
@@ -97,7 +98,12 @@ func Endpoints(ctx context.Context) *cobra.Command {
 				if manifest.Name != "" {
 					options.Name = manifest.Name
 				} else {
-					options.Name = utils.InferName(cwd)
+					c, _, err := okteto.NewK8sClientProvider().Provide(okteto.Context().Cfg)
+					if err != nil {
+						return err
+					}
+					// TODO: Check manifest. It changes, probably need to keep the one specified by the user
```

Comment:
i am not sure about this comment. if namespace is checked after... should be first check manifest namespace so options.Namespace has the one at the manifest? and then use this insead of okteto.Context().Namespace?

Reply:
After checking the logic, it seems ok as it is now, so I deleted the comment